execution_engine_testing/tests/src/test/regression/regression_20220221.rs

```diff
@@ -73,24 +63,21 @@ fn regression_20220221_should_distribute_to_many_validators() {
 
     builder.upgrade_with_upgrade_request_and_config(None, &mut upgrade_request);
 
-    builder.exec(fund_request).expect_success().commit();
+    builder.transfer_and_commit(fund_request).expect_success();
 
     // Add validators
     for _ in 0..DEFAULT_MAX_RUNTIME_CALL_STACK_HEIGHT {
         let public_key = public_keys.next().unwrap();
 
-        let transfer_request = ExecuteRequestBuilder::transfer(
-            *DEFAULT_ACCOUNT_ADDR,
-            runtime_args! {
-                mint::ARG_TARGET => public_key.to_account_hash(),
-                mint::ARG_AMOUNT => U512::from(MINIMUM_ACCOUNT_CREATION_BALANCE / 10),
-                mint::ARG_ID => <Option<u64>>::None,
-            },
+        let transfer_request = TransferRequestBuilder::new(
+            MINIMUM_ACCOUNT_CREATION_BALANCE / 10,
+            public_key.to_account_hash(),
         )
```

Comment:
why are we removing `.with_protocol_version(*NEW_PROTOCOL_VERSION)` here?

Reply:
There was a plan to remove the explicit passing of protocol version when calling the top-level EE execute methods, since that was a left-over from the days of the Scala node.  I had done some of that work (it's removed from `ExecuteRequest`), and had planned to do the same for native requests.  I haven't removed the protocol version from the other request types in this PR as they've been mutating during the lifespan of this PR.

I still think it's a worthwhile change, but again, I'd leave that for a follow-up PR if others agree.

If that's not agreed, we should probably check the usage of the protocol version in native transactions, as it appears it's ignored (as you spotted here - we're creating a transfer request with version 2.0.0 and successfully executing it on a 3.0.0 data-access layer).